setup.py

```diff
@@ -67,7 +67,7 @@ def get_version(_mod_root):
         if not src_root:
             src_root = '.'
 
-        with open(src_root + '/VERSION', 'r') as f:
+        with open(src_root + '/VERSION', 'r', encoding='utf-8') as f:
```

Comment:
There is an `ru.ru_open` which does the same to ensure we use the same encoding everywhere.

Reply:
but as I understand it is not the case for `setup.py` file (same for other packages)
if RU is not installed then installation will crash without trying to install its dependency